tools/codegen/gen.py

```diff
@@ -952,6 +953,32 @@ def write_outputs(self, filename: str) -> None:
             filename,
             ''.join(name + ";" for name in sorted(self.filenames)))
 
+def get_custom_build_selector(
+        provided_op_registration_allowlist: Optional[List[str]],
+        op_selection_yaml_path: Optional[str]) -> SelectiveBuilder:
+    assert not (
+        provided_op_registration_allowlist is not None and
+        op_selection_yaml_path is not None), (
+            "Both provided_op_registration_allowlist and " +
+            "op_selection_yaml_path can NOT be provided at the " +
+            "same time.")
+
+    op_registration_allowlist: Optional[Set[str]] = None
+    if provided_op_registration_allowlist is not None:
+        op_registration_allowlist = set(provided_op_registration_allowlist)
+
+    selector: SelectiveBuilder = SelectiveBuilder.get_nop_selector()
```

Comment:
nit: just assign selector = get_no_selector in the else branch

Reply:
Addressed in latest rev.